src/apache/bin/httpd-wrapper

```diff
@@ -25,6 +27,19 @@ else
 	echo "No certificates are active: using HTTP only"
 fi
 
+if should_enable_prettyurls; then
+    if is_htaccess_rewritebase_valid; then
```

Comment:
is maybe this the problem during a restart?
Can the function retreive any value from nextcloud during a restart?

Reply:
seems like this was the case. is fixed now.